Gem/Code/Source/Components/NetworkPlayerMovementComponent.cpp

```diff
@@ -200,6 +200,15 @@ namespace MultiplayerSample
         NetworkPlayerMovementComponentNetworkInput* playerInput = input.FindComponentInput<NetworkPlayerMovementComponentNetworkInput>();
         if (playerInput->m_resetCount != GetNetworkTransformComponentController()->GetResetCount())
         {
+            // TEMPORARY LOGGING TO HELP DIAGNOSE MOVEMENT DISCONNECTIONS.
+            // Specifically, sometimes a player will still be able to move around locally, but on the server and other connected clients,
+            // the player has stopped moving. They can no longer interact with the environment, but they are still connected and can see
+            // the other players moving around, still receive damage notifications,e tc.
+            // This logging will get removed after the root cause has been found and resolved.
+            AZLOG_INFO("netEntityId=%u: Different reset count, discarding player input. Input / local reset=%u / %u, clientInputId=%u, hostFrame=%u",
```

Comment:
When his happens do you see a non-stop corrections being sent to that client?

Reply:
I don't _think_ so, but I'm not 100% sure, since the only log I have is from a pretty long play session with a lot of corrections in it to begin with. But since this is the only early-return in ProcessInput(), this will at least tell us if we're hitting that pathway when the problem occurs, or else if the problem is occurring somewhere else in the chain (like in LocalPrediction...cpp or something).